app/views/teams/show.html.erb

```diff
@@ -1,32 +1,37 @@
 <p id="notice"><%= notice %></p>
 
-<p>
-  <% if @team.image_url %>
-  <%= image_tag @team.image_url %>
-  <% end %>
-</p>
-
-<p>
-  <strong>Name:</strong>
-  <%= @team.name %>
-</p>
-
-<p>
-<strong>Team Members</strong>
-</p>
-<table>
-  <tbody>
-    <% @team.users.each do |user| %>
-    <% @user = user %>
-    <tr>
-      <td><%= user.first_name %></td>
-      <td><%= user.last_name %></td>
-      <td><%= user.email %></td>
-      <td><%= user.score %></td>
-    </tr>
+<div class="row">
+
+<div id="showImage">
+    <% if @team.image_url %>
+    <%= image_tag @team.image_url %>
     <% end %>
-  </tbody>
-</table>
+</div>
+
+<div id="showTable">
+
+  <table>
+    <caption>
+      <h2><%= @team.name %></h3>
+    </caption>
+
+    <tbody>
+      <% @team.users.each do |user| %>
+      <% @user = user %>
+      <tr>
+        <td><%= user.first_name %></td>
+        <td><%= user.last_name %></td>
+        <td><%= user.email %></td>
+        <td><%= user.score %></td>
+      </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <div id="teamButs">
+    <%= link_to 'Leave team', leave_team_path(@team), :class => 'button', :id => 'warning' %>
```

Comment:
@veddster Here you can use the Ruby 2.0 Hash syntax, it's the same only to use the new syntax:

`<%= link_to 'Leave team', leave_team_path(@team), class: 'button', id: 'warning' %>`